src/main.browser.ts

```diff
@@ -0,0 +1,8 @@
+import 'zone.js/dist/zone';
+import 'reflect-metadata';
+import 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { BrowserAppModule } from './app/browser-app.module';
+
+platformBrowserDynamic().bootstrapModule(BrowserAppModule);
```

Comment:
Yes - Please use platformBrowser and AOT-ed factory instead of platformBrowserDynamic. And you shouldn't need reflect-metadata as well in that case.